lib/auth/kakao.ts

```
@@ -1,15 +1,51 @@
+import { setCookie } from 'cookies-next';
 import { AuthError } from '@/lib/error';
+import { generateRandomNonce, generateRandomString, hashNonce } from '.';
 
-export function kakaoLogin(redirectUri?: string) {
+function initializeKakao() {
+  if (!window.Kakao) {
+    throw new Error('Kakao SDK is not loaded.');
+  }
+
+  if (!window.Kakao.isInitialized()) {
+    window.Kakao.init(process.env.NEXT_PUBLIC_JAVASCRIPT_KEY!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)